google_project/locals.tf

```diff
@@ -32,4 +32,7 @@ locals {
     "stackdriver.googleapis.com",
   ]
   all_project_services = setunion(local.default_project_services, var.project_services)
+
+  default_data_access_logs = ["iam.googleapis.com", "secretmanager.googleapis.com", "sts.googleapis.com"]
```

Comment:
Not binding and out of curiosity: Do we also need cloudkms to get the key accesses under secretmanager for things like the hsm slot updates and such when autograph is doing that work? (I'm happy for us to do that later when we actually need it)